client/package.json

```diff
@@ -13,7 +13,7 @@
     "react": "^16.13.1",
     "react-bootstrap": "^1.0.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "^3.4.4",
+    "react-scripts": "^4.0.3",
```

Comment:
why this change tho?

Reply:
These changes were automatically been made while making the docker file